core/domain/expression_parser_test.py

```diff
@@ -59,6 +59,7 @@ def test_is_algebraic(self):
         self.assertTrue(expression_parser.is_algebraic('a^2.3'))
         self.assertTrue(expression_parser.is_algebraic('abs(alpha)'))
         self.assertTrue(expression_parser.is_algebraic('alpha/gamma'))
+        self.assertTrue(expression_parser.is_algebraic('alpha + bet/22'))
```

Comment:
Wait, this shouldn't be algebraic, right? Since `bet` is not a valid identifier?

Reply:
Yup, sorry, just saw that. Fixed it now.